cmake/pkgconfig/ignition.pc.in

```diff
@@ -1,4 +1,4 @@
-prefix=@CMAKE_INSTALL_PREFIX@
+prefix=${pcfiledir}/../..
```

Comment:
as discussed in dartsim/dart#1527 (comment), this relative path is incorrect on Ubuntu

Reply:
I've updated the `.pc.in` templates for downstream ignition packages in 1e39fa0 and the `.pc` file for ign-cmake2 itself in b95d20f